### PR TITLE
Feature/blui upgrade docs var

### PR DESCRIPTION
**Pull request by shubham-eaton**
Fixes:
- Fixed docs tab text issue

#### Changes proposed in this Pull Request:

- Used var syntax of mui 6 in docs
-
-

#### Screenshots / Screen Recording (if applicable)

-
Screenshot 2024-12-19 at 3 29 54 PM

#### To Test:

-

#### Any specific feedback you are looking for?

-

### CODE REVIEW

**Review by ektaghag-eaton — APPROVED** [review]

File: docs/src/shared/CallToActionButton.tsx
Comment (surajeaton) [inline docs/src/shared/CallToActionButton.tsx:1]: You can use mode from useColorScheme in line 76.

File: docs/src/App.tsx
Comment (surajeaton) [inline docs/src/App.tsx:1]: You can use mode from useColorScheme in line 76 in Themeexplorer

File: docs/src/shared/CallToActionButton.tsx
Comment (shubham-eaton) [reply docs/src/shared/CallToActionButton.tsx:1]: raised PR https://github.com/etn-ccis/blui-react-component-library/pull/948

File: docs/src/App.tsx
Comment (shubham-eaton) [reply docs/src/App.tsx:1]: raised PR https://github.com/etn-ccis/blui-react-component-library/pull/948

**Pull request merged**